examples/erc1155/lib.rs

```diff
@@ -341,7 +350,11 @@ mod erc1155 {
                 value,
             });
 
-            // Quick Haxx, otherwise my tests just panic due to the use of eval_contract()
+            // This is disabled during tests due to the use of `eval_contract()` not being
+            // supported (tests end up panicking).
+            //
+            // We should be able to get rid of this with when the new off-chain testing
+            // environment is available.
```

Comment:
The new off-chain environment won't support this either unfortunately. It is pretty hard to support in general.